test/Project.toml

```
@@ -4,5 +4,5 @@ Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-Documenter = "0.27"
+Documenter = "1"
```

It seems Julia 1.0 is not compatible with Documenter >= 1.0. It would be fine to drop support for Julia 1.0 at this point but I think we can keep it for now since it should not require any additional maintenance: For the doctests, docs/make.jl and the Documenter 1.0 specific settings are irrelevant, so keeping support for Documenter 0.27 in the tests seems safe.
```suggestion
Documenter = "0.27, 1"
```